bots/jabber_mirror_backend.py

```diff
@@ -144,6 +155,7 @@ def message(self, msg):
             logging.exception("Error forwarding Jabber => Zulip")
 
     def private(self, msg):
+        # type: (Any) -> None
```

Comment:
Yes, but as you can see in the above `message` method, `msg` gets passed to `logging.warning` and that does not accept a dict as argument so it can't be a dict.